tests/performance-tests/generate_test_config.rb

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env ruby
+
+require 'json'
+require 'yaml'
+
+manifest_file = ENV.fetch("CF_MANIFEST")
+manifest = YAML.load_file(manifest_file)
+
+admin_password = manifest.fetch("properties").fetch("performance_tests").fetch("admin_password")
+api_url = manifest.fetch("properties").fetch("cc").fetch("srv_api_uri")
```

Comment:
I think you can grab this from properties/acceptance_tests - as you are building on acceptance test framework.
You can then remove whole 'performance tests' part from manifest, as you don't need it - and nothing else uses it.